include/boost/url/detail/config.hpp

```diff
@@ -12,7 +12,8 @@
 
 #include <boost/config.hpp>
 #include <boost/config/workaround.hpp>
-#include <limits.h>
+#include <climits>
+#include <cstdint>
```

Comment:
I really try to keep the number of includes in this file down to a minimum, because it is included everywhere. That's why I included limits.h instead of cstdint and climits.